src/Exadel.CrazyPrice/Exadel.CrazyPrice.Tests/Data/Extentions/DbUserExtentionsTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Exadel.CrazyPrice.Common.Models;
+using Exadel.CrazyPrice.Data.Extentions;
+using Exadel.CrazyPrice.Data.Models;
+using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Exadel.CrazyPrice.Tests.Data.Extentions
+{
+    public class DbUserExtentionsTests
+    {
+        [Fact]
+        public void GetOneFreeDbUserTest()
+        {
+            var users = new List<DbUser>();
+            users.GetOne().Should().BeEquivalentTo(new DbUser());
+        }
+
+        [Fact]
+        public void ToUserTest()
+        {
+            var user = new DbUser();
+            user.ToUser().Should().BeEquivalentTo(new User());
+        }
+
+        [Fact]
+        public void ToDbUserTest()
+        {
+            var user = new User()

```

Comment:
Test all fields here as Theory

Reply:
[Added field.](https://github.com/yegorl/exadel_discounts_be/pull/24/commits/db1d32b198801a29e75ba7c90f1c2facff55e39f)
 Theory not needed because logic is not in class.